src/CLI/Version.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Arkitect\CLI;
+
+class Version
+{
+    private const COMPOSER_PATH = __DIR__.'/../../composer.json';
```

Comment:
does it also works on the phar version?

Reply:
I have added more possible paths for composer.json file